examples/src/test/java/ai/djl/examples/training/TrainResNetTest.java

```diff
@@ -48,7 +48,7 @@ public void testTrainResNetSymbolicNightly()
         if (Device.getGpuCount() > 0) {
             // Limit max 4 gpu for cifar10 training to make it converge faster.
             // and only train 10 batch for unit test.
-            String[] args = {"-e", "10", "-g", "4", "-s", "-p"};
+            String[] args = {"-e", "15", "-g", "4", "-s", "-p"};
```

Comment:
We should lower the thresholds rather than raise the number of epochs. We want our tests to run faster if possible